<!DOCTYPE html><html>
<head>
<title>organismic supercategories and super-complex systems biodynamics</title>
<!--Generated on Fri Feb  9 19:30:10 2018 by LaTeXML (version 0.8.2) http://dlmf.nist.gov/LaTeXML/.-->

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<link rel="stylesheet" href="LaTeXML.css" type="text/css">
<link rel="stylesheet" href="ltx-article.css" type="text/css">
<link rel="stylesheet" href="https://cdn.rawgit.com/holtzermann17/3f71ceeb3b055e1ddc3b6c11fb1f074c/raw/2bb23e3b173ff96840797fc0c3bcb8c54085df8e/LaTeXML.css" type="text/css">
<link rel="stylesheet" href="https://cdn.rawgit.com/holtzermann17/4bda0365b30858ac2fb83623185fe3ec/raw/cedd84ed3e3ad597c5d293f443ecfe4803741c6b/ltx-article.css" type="text/css">
<script src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=MML_HTMLorMML" type="text/javascript"></script>
</head>
<body>
<div class="ltx_page_main">
<div class="ltx_page_content">
<article class="ltx_document ltx_authors_1line">
<h1 class="ltx_title ltx_title_document">organismic supercategories and super-complex systems biodynamics</h1>

<div id="p1" class="ltx_para">
<br class="ltx_break">
</div>
<section id="S0.SS1" class="ltx_subsection">
<h2 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">0.1 </span>Organismic Supercategories and Complex Systems Biodynamics</h2>

<section id="S0.SS1.SSS1" class="ltx_subsubsection">
<h3 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">0.1.1 </span>A Dynamic System Example: Connecting Topological and Algebraic Structures</h3>

<div id="S0.SS1.SSS1.p1" class="ltx_para">
<p class="ltx_p">Let us consider an example of a system whose state space consists of a torus, <math id="S0.SS1.SSS1.p1.m1" class="ltx_Math" alttext="T^{3}" display="inline"><msup><mi>T</mi><mn>3</mn></msup></math>, such that the states of the system are contained inside the torus, and all transitions lead to states inside the torus. The homology theory offers in this case two intuitive examples of generators as the entire torus is generated by only two cycles.
These two cycles generate two homology groups <math id="S0.SS1.SSS1.p1.m2" class="ltx_Math" alttext="H_{0}(T)=Z" display="inline"><mrow><mrow><msub><mi>H</mi><mn>0</mn></msub><mo>⁢</mo><mrow><mo stretchy="false">(</mo><mi>T</mi><mo stretchy="false">)</mo></mrow></mrow><mo>=</mo><mi>Z</mi></mrow></math> and <math id="S0.SS1.SSS1.p1.m3" class="ltx_Math" alttext="H_{1}(T)=Z\otimes Z" display="inline"><mrow><mrow><msub><mi>H</mi><mn>1</mn></msub><mo>⁢</mo><mrow><mo stretchy="false">(</mo><mi>T</mi><mo stretchy="false">)</mo></mrow></mrow><mo>=</mo><mrow><mi>Z</mi><mo>⊗</mo><mi>Z</mi></mrow></mrow></math>, where <math id="S0.SS1.SSS1.p1.m4" class="ltx_Math" alttext="Z" display="inline"><mi>Z</mi></math> is the group of integers, and <math id="S0.SS1.SSS1.p1.m5" class="ltx_Math" alttext="\otimes" display="inline"><mo>⊗</mo></math> denotes the product operation. These homology groups give a characterization of the topological space represented by the torus. In this way a connection is established between a <em class="ltx_emph ltx_font_italic">topological</em> structure-that of the toroidal space, and <em class="ltx_emph ltx_font_italic">algebraic</em> structures, those of homology groups, for example. Even more, we can assign two numbers to a given complex <math id="S0.SS1.SSS1.p1.m6" class="ltx_Math" alttext="K" display="inline"><mi>K</mi></math> : the Betti number-which is the number of repetitions of <math id="S0.SS1.SSS1.p1.m7" class="ltx_Math" alttext="Z" display="inline"><mi>Z</mi></math> in the homology group,
<math id="S0.SS1.SSS1.p1.m8" class="ltx_Math" alttext="H_{p}(K)=Z\otimes Z\otimes...\otimes Z\otimes G_{PT}" display="inline"><mrow><mrow><msub><mi>H</mi><mi>p</mi></msub><mo>⁢</mo><mrow><mo stretchy="false">(</mo><mi>K</mi><mo stretchy="false">)</mo></mrow></mrow><mo>=</mo><mrow><mi>Z</mi><mo>⊗</mo><mi>Z</mi><mo>⊗</mo><mi mathvariant="normal">…</mi><mo>⊗</mo><mi>Z</mi><mo>⊗</mo><msub><mi>G</mi><mrow><mi>P</mi><mo>⁢</mo><mi>T</mi></mrow></msub></mrow></mrow></math> of a complex <math id="S0.SS1.SSS1.p1.m9" class="ltx_Math" alttext="K" display="inline"><mi>K</mi></math>-and a number <math id="S0.SS1.SSS1.p1.m10" class="ltx_Math" alttext="p" display="inline"><mi>p</mi></math> which is the number of elements of a finite abelian group <math id="S0.SS1.SSS1.p1.m11" class="ltx_Math" alttext="G_{PT}" display="inline"><msub><mi>G</mi><mrow><mi>P</mi><mo>⁢</mo><mi>T</mi></mrow></msub></math>. The Betti number gives the number of <math id="S0.SS1.SSS1.p1.m12" class="ltx_Math" alttext="p" display="inline"><mi>p</mi></math>-dimensional holes of the complex <math id="S0.SS1.SSS1.p1.m13" class="ltx_Math" alttext="K" display="inline"><mi>K</mi></math>, and the number <math id="S0.SS1.SSS1.p1.m14" class="ltx_Math" alttext="p" display="inline"><mi>p</mi></math> gives the number of <math id="S0.SS1.SSS1.p1.m15" class="ltx_Math" alttext="p" display="inline"><mi>p</mi></math>-dimensional turns of <math id="S0.SS1.SSS1.p1.m16" class="ltx_Math" alttext="K" display="inline"><mi>K</mi></math>.
A natural physical interpretation that one can give to the holes inside of the state space of a system will be that of <em class="ltx_emph ltx_font_italic">instability fields</em> of the system under consideration. Consequently, the Betti number will give a coarse idea about the instability of the system, being the number of instability fields of the state space. However, homological as well as homotopy techniques would allow a much finer characterization of the local and global properties of the dynamics of a system, being able, for example, to locate singularities in a state space (Hwa and Teplitz, 1966). It must be mentioned here that the theory of categories and functors provides a natural framework for homological techniques, mostly for abelian structures in commutative homological algebra. In the example discussed above, a number was assigned to a quality, that is, a Betti number was assigned to a topological space. Another example of such an assignment is found in the theory of elementary particles, where one associates a probability with a Feynman diagram
that qualitatively represents particle interactions <em class="ltx_emph ltx_font_italic">via</em> quantum fields or exchanged quanta.
However, the general procedure is not to assign a single number to a quality, but an entire set of numbers or elements. In our second example the operation of addition induced a corresponding operation on diagrams. This fact suggests that operations which are used in metric, or quantitative, biology may induce corresponding operations in relational biology. Conversely, one can think of significant relational operations with notions which would permit us to obtain solutions of complicated problems of quantitative biology.</p>
</div>
</section>
<section id="S0.SS1.SSS2" class="ltx_subsubsection">
<h3 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">0.1.2 </span>Observables, Generators and Qualitative Dynamics.</h3>

<div id="S0.SS1.SSS2.p1" class="ltx_para">
<p class="ltx_p">Observables of a biological system may be introduced as intensities of some
(metabolic / biochemical / enzymatic, or genetic, etc.) <em class="ltx_emph ltx_font_italic">activities</em> of the living system (as suggested for example by Rashevsky and Rosen); alternatively, they may be considered either as <em class="ltx_emph ltx_font_italic">parameters</em> characterizing specific or global processes inside the system, or as <em class="ltx_emph ltx_font_italic">variables</em> (e.g., molecular set variables) that specify the quantities of certain bioproducts which are formed as a result of the activities of the system. Among observables,
<em class="ltx_emph ltx_font_italic">structural parameters</em> as defined in (Rosen 1968a, b) and <em class="ltx_emph ltx_font_italic">time observables</em> play distinguished roles. Some observables are “linked”, in the sense that, a change in one of the observables implies a corresponding change of the others. <em class="ltx_emph ltx_font_italic">Linked observables</em> were represented as <em class="ltx_emph ltx_font_italic">morphisms</em> in a categorical diagram. Such a diagram may represent, for example, the linkage group of observables and is a part of the <em class="ltx_emph ltx_font_italic">generating class</em> of the system. Since diagrams can be regarded as functors (Mitchell, 1965) one can represent the linked observables that specify the generating classes of a system by <em class="ltx_emph ltx_font_italic">‘dynamic configuration’ functors</em> between categories of possible system configurations at different times; therefore, qualitative changes in system dynamics may be represented by natural transformations of such dynamic configuration functors, whereas transition-state morphisms assign numerical values to observables in different system states. There is then a classification problem associated with defining, or determining through measurements the biodynamics of super-complex systems as the latter are endowed with <em class="ltx_emph ltx_font_italic">variable state-space (or configuration space) topologies</em> (Baianu et al, 2007a). This leads unavoidably to the consideration of supercategories of variable topological structures and their associated algebroids related to global stability and qualitative dynamics.</p>
</div>
<div id="S0.SS1.SSS2.p2" class="ltx_para">
<p class="ltx_p">Let us define a state of a system or organismic set, biomolecular set, <math id="S0.SS1.SSS2.p2.m1" class="ltx_Math" alttext="(M,R)" display="inline"><mrow><mo stretchy="false">(</mo><mi>M</mi><mo>,</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></math>-system,…, any theoretical model of a biosystem, at a given moment as an <math id="S0.SS1.SSS2.p2.m2" class="ltx_Math" alttext="n" display="inline"><mi>n</mi></math>-tuple of the values of essential observables at that moment. In this representation, either a state, or system ‘configuration’, is then defined by a dynamic functor from the category of generating classes of the system to <math id="S0.SS1.SSS2.p2.m3" class="ltx_Math" alttext="mathsf{\mathcal{R}}" display="inline"><mrow><mi>m</mi><mo>⁢</mo><mi>a</mi><mo>⁢</mo><mi>t</mi><mo>⁢</mo><mi>h</mi><mo>⁢</mo><mi>s</mi><mo>⁢</mo><mi>f</mi><mo>⁢</mo><mi class="ltx_font_mathcaligraphic">ℛ</mi></mrow></math>–the set of real numbers regarded for example as a discrete category (or as a category whose objects are real numbers, and whose morphisms are mappings; the operations with real numbers in this category are induced by the structure of the category of generating classes). Let us also consider here a specific example from microbial genetics. An <em class="ltx_emph ltx_font_italic">operon</em> as defined by Jacob and Monod (1961) may be considered- in a very simple model- as having two states: an active state and an inactive one. In its inactive state the operon will not induce the synthesis of the corresponding enzyme, while in its active state, it will induce the synthesis of a determined quantity <math id="S0.SS1.SSS2.p2.m4" class="ltx_Math" alttext="C" display="inline"><mi>C</mi></math> of synthesized enzyme per unit of time. Now, if we consider a linkage group of operons
<math id="S0.SS1.SSS2.p2.m5" class="ltx_Math" alttext="O_{1},O_{2},...,O_{n}" display="inline"><mrow><msub><mi>O</mi><mn>1</mn></msub><mo>,</mo><msub><mi>O</mi><mn>2</mn></msub><mo>,</mo><mi mathvariant="normal">…</mi><mo>,</mo><msub><mi>O</mi><mi>n</mi></msub></mrow></math>, which are all active in the same time and, if the synthesized quantities of enzymes per unit of time are respectively, <math id="S0.SS1.SSS2.p2.m6" class="ltx_Math" alttext="C_{1},C_{2},...,C_{n}" display="inline"><mrow><msub><mi>C</mi><mn>1</mn></msub><mo>,</mo><msub><mi>C</mi><mn>2</mn></msub><mo>,</mo><mi mathvariant="normal">…</mi><mo>,</mo><msub><mi>C</mi><mi>n</mi></msub></mrow></math>, a state of the linked operons may be defined by the <math id="S0.SS1.SSS2.p2.m7" class="ltx_Math" alttext="n" display="inline"><mi>n</mi></math>-tuple <math id="S0.SS1.SSS2.p2.m8" class="ltx_Math" alttext="(C^{t}_{1},C^{t}_{2},...,C^{t}_{n})" display="inline"><mrow><mo stretchy="false">(</mo><msubsup><mi>C</mi><mn>1</mn><mi>t</mi></msubsup><mo>,</mo><msubsup><mi>C</mi><mn>2</mn><mi>t</mi></msubsup><mo>,</mo><mi mathvariant="normal">…</mi><mo>,</mo><msubsup><mi>C</mi><mi>n</mi><mi>t</mi></msubsup><mo stretchy="false">)</mo></mrow></math> of the values of <math id="S0.SS1.SSS2.p2.m9" class="ltx_Math" alttext="C_{1},C_{2},...,C_{n}" display="inline"><mrow><msub><mi>C</mi><mn>1</mn></msub><mo>,</mo><msub><mi>C</mi><mn>2</mn></msub><mo>,</mo><mi mathvariant="normal">…</mi><mo>,</mo><msub><mi>C</mi><mi>n</mi></msub></mrow></math> at a given instant <math id="S0.SS1.SSS2.p2.m10" class="ltx_Math" alttext="t" display="inline"><mi>t</mi></math>. However, suppose that only <math id="S0.SS1.SSS2.p2.m11" class="ltx_Math" alttext="C_{1},C_{2}" display="inline"><mrow><msub><mi>C</mi><mn>1</mn></msub><mo>,</mo><msub><mi>C</mi><mn>2</mn></msub></mrow></math> and <math id="S0.SS1.SSS2.p2.m12" class="ltx_Math" alttext="C_{3}" display="inline"><msub><mi>C</mi><mn>3</mn></msub></math> are essential, and all other observables can be expressed in terms of <math id="S0.SS1.SSS2.p2.m13" class="ltx_Math" alttext="C_{1},C_{2}" display="inline"><mrow><msub><mi>C</mi><mn>1</mn></msub><mo>,</mo><msub><mi>C</mi><mn>2</mn></msub></mrow></math> and <math id="S0.SS1.SSS2.p2.m14" class="ltx_Math" alttext="C_{3}" display="inline"><msub><mi>C</mi><mn>3</mn></msub></math>. Moreover, let us suppose that we may find some operators such that <math id="S0.SS1.SSS2.p2.m15" class="ltx_Math" alttext="C_{2}=YC_{1}" display="inline"><mrow><msub><mi>C</mi><mn>2</mn></msub><mo>=</mo><mrow><mi>Y</mi><mo>⁢</mo><msub><mi>C</mi><mn>1</mn></msub></mrow></mrow></math> and <math id="S0.SS1.SSS2.p2.m16" class="ltx_Math" alttext="C_{3}=ZC_{2}" display="inline"><mrow><msub><mi>C</mi><mn>3</mn></msub><mo>=</mo><mrow><mi>Z</mi><mo>⁢</mo><msub><mi>C</mi><mn>2</mn></msub></mrow></mrow></math>. In this case there exists a third operator <math id="S0.SS1.SSS2.p2.m17" class="ltx_Math" alttext="X" display="inline"><mi>X</mi></math> such that <math id="S0.SS1.SSS2.p2.m18" class="ltx_Math" alttext="C_{3}=XC_{1}" display="inline"><mrow><msub><mi>C</mi><mn>3</mn></msub><mo>=</mo><mrow><mi>X</mi><mo>⁢</mo><msub><mi>C</mi><mn>1</mn></msub></mrow></mrow></math>, which makes the three operon diagram commutative.
This is the <em class="ltx_emph ltx_font_italic">generating diagram</em> of the linkage group of operators. In this simple model, a <em class="ltx_emph ltx_font_italic">genetic system</em>,
or organsimic set (discrete organismic supercategory) of zero-th order, <math id="S0.SS1.SSS2.p2.m19" class="ltx_Math" alttext="S_{0}" display="inline"><msub><mi>S</mi><mn>0</mn></msub></math>, will be then represented by a <em class="ltx_emph ltx_font_italic">generating class</em>, whose objects are generating diagrams of the linkage groups of operons, and whose morphisms are the functional connections among the activities of the operons. Suppose that a mutation takes place in such a genetic system, so that an operon will begin to induce the synthesis of an enzyme which was not induced previously by the genetic system. The state corresponding to the very moment when the change takes place will be considered as a singularity of the state space, because at that moment when the genetic mutation occurred one cannot characterize the state of the genetic system either by <math id="S0.SS1.SSS2.p2.m20" class="ltx_Math" alttext="C^{t}_{k}" display="inline"><msubsup><mi>C</mi><mi>k</mi><mi>t</mi></msubsup></math> –the quantity of synthesized enzyme <math id="S0.SS1.SSS2.p2.m21" class="ltx_Math" alttext="E_{k}" display="inline"><msub><mi>E</mi><mi>k</mi></msub></math> per unit of time, or by <math id="S0.SS1.SSS2.p2.m22" class="ltx_Math" alttext="C_{k}" display="inline"><msub><mi>C</mi><mi>k</mi></msub></math> the quantity of the ‘mutated’ enzyme <math id="S0.SS1.SSS2.p2.m23" class="ltx_Math" alttext="E" display="inline"><mi>E</mi></math> per unit of time (that is, <math id="S0.SS1.SSS2.p2.m24" class="ltx_Math" alttext="E" display="inline"><mi>E</mi></math> is the enzyme which begins to be synthesized only after the mutation took place). It may happen that a mutation produces effects such as the complete inactivation of an essential operon. In this case, it is conceivable that the whole linkage group will become inactive. If the inactivated operon is the <em class="ltx_emph ltx_font_italic">replicon</em> (Jacob et al., 1963), then the cell will cease to divide, and eventually die. The singularity of the genomic state space in the case of a mutation would last much less than other states of the dynamical system, and may be therefore considered as an <em class="ltx_emph ltx_font_italic">unstable</em> state inside the state space of the system.</p>
</div>
<div id="S0.SS1.SSS2.p3" class="ltx_para">
<p class="ltx_p">Generally, if the unstable state leads only to other unstable states, it may result in the destruction of the system-generating an <em class="ltx_emph ltx_font_italic">unstable field</em> outside the state space stable attractors of the system. Consequently, states in growth processes should have to be considered as <em class="ltx_emph ltx_font_italic">metastable</em>, and cannot be simply considered as unstable. The replacement of an observable by another, in the case of a mutation discussed here, is in fact a change of the structure of the genetic system– a structure which has both algebraic and topological representations in a <em class="ltx_emph ltx_font_italic">supercatgeory</em>,
that may also be viewed as a family or class of multiple, <em class="ltx_emph ltx_font_italic">(variable) categories</em>; the latter type of generalized structure necessary for the representation of super-complex system biodynamics will be defined more precisely in the
next section as an interpretation of <em class="ltx_emph ltx_font_italic">ETAS</em> axioms, a natural extension of Lawvere’s <em class="ltx_emph ltx_font_italic">ETAC</em> axioms for the foundation of the theory of categories and functors.</p>
</div>
</section>
</section>
<section id="S0.SS2" class="ltx_subsection">
<h2 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">0.2 </span>Biodynamics in Organismic Supercategories</h2>

<div id="S0.SS2.p1" class="ltx_para">
<p class="ltx_p">Axiomatic definitions of categories and supercategories based respectively on <em class="ltx_emph ltx_font_italic">ETAC</em> and <em class="ltx_emph ltx_font_italic">ETAS</em> interprertations provide a framework for representations of Super- Complex Biological Systems and also allow for dynamic computations of cell transformations that may lead to neoplasia, and in certain intriguing cases to malignancy. The concepts of quantum automata and quantum computation are applied in the context of quantum genetics and genetic networks to study their nonlinear dynamics. In a previous publication (Baianu,1971a) the formal concept of quantum automaton was introduced and its possible implications for genetic and metabolic activities in living cells and organisms were considered. This was followed by a report on quantum and abstract, symbolic computation based on the theory of categories, functors and natural transformations (Baianu,1971b). The notions of topological semigroup, quantum automaton, or quantum computer, were then suggested with a view to their potential applications to the analogous simulation of biological systems, and especially genetic activities and nonlinear dynamics in genetic networks. Further, detailed studies of nonlinear dynamics in genetic networks were carried out in
categories of <math id="S0.SS2.p1.m1" class="ltx_Math" alttext="n" display="inline"><mi>n</mi></math>-valued, Łukasiewicz Logic Algebras (<span class="ltx_text ltx_font_typewriter">http://planetmath.org/AlgebraicCategoryOfLMnAlgebras</span>) that showed significant dissimilarities (Baianu, 1977; Baianu et al., 2006-2008) from the oversimplified Boolean models of human neural networks.A categorical and Topos framework for Lukasiewicz Algebraic Logic models of nonlinear dynamics in complex functional genomes and cell interactomes is proposed. Lukasiewicz Algebraic Logic models of genetic networks and signaling pathways in cells are formulated in terms of nonlinear dynamic systems with n-state components that allow for the generalization of previous logical models of both genetic activities and neural networks. An algebraic formulation of variable ‘next-state functions’ is extended to a Łukasiewicz Topos endowed with a <math id="S0.SS2.p1.m2" class="ltx_Math" alttext="n" display="inline"><mi>n</mi></math>-valued
Łukasiewicz algebraic logic subobject classifier description that represents non-random and nonlinear network activities as well as their transformations in developmental processes and carcinogenesis. Novel results and specific applications concerning cell interactomics, dynamics of genetic-proteomic networks and signaling pathways, development, regeneration, the control mechanisms of cell dynamic programming in cells, neoplastic transformations and oncogenesis are then derived on the basis of complex system modeling and biomolecular network representations in categories of
Łukasiewicz logic algebras and Łukasiewicz-topos. Molecular models in terms of categories, functors and natural transformations were then formulated for unimolecular chemical transformations, as well as multi-molecular chemical and biochemical transformations (Baianu, 1983,2004a). Previous applications of computer modeling, classical automata theory, and relational biology to molecular biology, oncogenesis and medicine were extensively reviewed (Baianu,1987). Novel approaches to solving the realization problems of relational biology models in complex system biology are introduced in terms of natural transformations between functors of such biomolecular supercategories. Natural transformations of organismic superstructure were developed for modelling protein biosynthesis, embryogenesis and nuclear transplant experiments. Other possible realizations in molecular biology and relational biology of organisms are here suggested as a novel approach to bioinformatics to interactomics and relational quantum genetics.
</p>
<table class="ltx_tabular ltx_align_right ltx_align_middle">
<tbody class="ltx_tbody">
<tr class="ltx_tr">
<td class="ltx_td ltx_align_left ltx_border_l ltx_border_t">Title</td>
<td class="ltx_td ltx_align_left ltx_border_r ltx_border_t">organismic supercategories and super-complex systems biodynamics</td>
</tr>
<tr class="ltx_tr">
<td class="ltx_td ltx_align_left ltx_border_l">Canonical name</td>
<td class="ltx_td ltx_align_left ltx_border_r">OrganismicSupercategoriesAndSupercomplexSystemsBiodynamics</td>
</tr>
<tr class="ltx_tr">
<td class="ltx_td ltx_align_left ltx_border_l">Date of creation</td>
<td class="ltx_td ltx_align_left ltx_border_r">2013-03-22 18:16:57</td>
</tr>
<tr class="ltx_tr">
<td class="ltx_td ltx_align_left ltx_border_l">Last modified on</td>
<td class="ltx_td ltx_align_left ltx_border_r">2013-03-22 18:16:57</td>
</tr>
<tr class="ltx_tr">
<td class="ltx_td ltx_align_left ltx_border_l">Owner</td>
<td class="ltx_td ltx_align_left ltx_border_r">bci1 (20947)</td>
</tr>
<tr class="ltx_tr">
<td class="ltx_td ltx_align_left ltx_border_l">Last modified by</td>
<td class="ltx_td ltx_align_left ltx_border_r">bci1 (20947)</td>
</tr>
<tr class="ltx_tr">
<td class="ltx_td ltx_align_left ltx_border_l">Numerical id</td>
<td class="ltx_td ltx_align_left ltx_border_r">39</td>
</tr>
<tr class="ltx_tr">
<td class="ltx_td ltx_align_left ltx_border_l">Author</td>
<td class="ltx_td ltx_align_left ltx_border_r">bci1 (20947)</td>
</tr>
<tr class="ltx_tr">
<td class="ltx_td ltx_align_left ltx_border_l">Entry type</td>
<td class="ltx_td ltx_align_left ltx_border_r">Topic</td>
</tr>
<tr class="ltx_tr">
<td class="ltx_td ltx_align_left ltx_border_l">Classification</td>
<td class="ltx_td ltx_align_left ltx_border_r">msc 18D25</td>
</tr>
<tr class="ltx_tr">
<td class="ltx_td ltx_align_left ltx_border_l">Classification</td>
<td class="ltx_td ltx_align_left ltx_border_r">msc 18-00</td>
</tr>
<tr class="ltx_tr">
<td class="ltx_td ltx_align_left ltx_border_l">Classification</td>
<td class="ltx_td ltx_align_left ltx_border_r">msc 81P05</td>
</tr>
<tr class="ltx_tr">
<td class="ltx_td ltx_align_left ltx_border_l">Classification</td>
<td class="ltx_td ltx_align_left ltx_border_r">msc 81Q05</td>
</tr>
<tr class="ltx_tr">
<td class="ltx_td ltx_align_left ltx_border_l">Classification</td>
<td class="ltx_td ltx_align_left ltx_border_r">msc 92B05</td>
</tr>
<tr class="ltx_tr">
<td class="ltx_td ltx_align_left ltx_border_l">Synonym</td>
<td class="ltx_td ltx_align_left ltx_border_r">super-complex system biodynamics</td>
</tr>
<tr class="ltx_tr">
<td class="ltx_td ltx_align_left ltx_border_l">Related topic</td>
<td class="ltx_td ltx_align_left ltx_border_r">SupercategoriesOfComplexSystems</td>
</tr>
<tr class="ltx_tr">
<td class="ltx_td ltx_align_left ltx_border_l">Related topic</td>
<td class="ltx_td ltx_align_left ltx_border_r">GroupoidCDynamicalSystem</td>
</tr>
<tr class="ltx_tr">
<td class="ltx_td ltx_align_left ltx_border_l">Defines</td>
<td class="ltx_td ltx_align_left ltx_border_r">system dynamics</td>
</tr>
<tr class="ltx_tr">
<td class="ltx_td ltx_align_left ltx_border_l">Defines</td>
<td class="ltx_td ltx_align_left ltx_border_r">complex systems biodynamics</td>
</tr>
<tr class="ltx_tr">
<td class="ltx_td ltx_align_left ltx_border_l">Defines</td>
<td class="ltx_td ltx_align_left ltx_border_r">configuration space topology and algebra</td>
</tr>
<tr class="ltx_tr">
<td class="ltx_td ltx_align_left ltx_border_l">Defines</td>
<td class="ltx_td ltx_align_left ltx_border_r">dynamic system examples</td>
</tr>
<tr class="ltx_tr">
<td class="ltx_td ltx_align_left ltx_border_l">Defines</td>
<td class="ltx_td ltx_align_left ltx_border_r">qualitative dynamics</td>
</tr>
<tr class="ltx_tr">
<td class="ltx_td ltx_align_left ltx_border_l">Defines</td>
<td class="ltx_td ltx_align_left ltx_border_r">dynamic configuration functors</td>
</tr>
<tr class="ltx_tr">
<td class="ltx_td ltx_align_left ltx_border_l">Defines</td>
<td class="ltx_td ltx_align_left ltx_border_r">qualitative biodynamics</td>
</tr>
<tr class="ltx_tr">
<td class="ltx_td ltx_align_left ltx_border_l">Defines</td>
<td class="ltx_td ltx_align_left ltx_border_r">super-complex system</td>
</tr>
<tr class="ltx_tr">
<td class="ltx_td ltx_align_left ltx_border_l">Defines</td>
<td class="ltx_td ltx_align_left ltx_border_r">generating class</td>
</tr>
<tr class="ltx_tr">
<td class="ltx_td ltx_align_left ltx_border_b ltx_border_l">Defines</td>
<td class="ltx_td ltx_align_left ltx_border_b ltx_border_r">dynamic multi-stability</td>
</tr>
</tbody>
</table>
</div>
</section>
</article>
</div>
<footer class="ltx_page_footer">
<div class="ltx_page_logo">Generated  on Fri Feb  9 19:30:10 2018 by <a href="http://dlmf.nist.gov/LaTeXML/">LaTeXML <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAsAAAAOCAYAAAD5YeaVAAAAAXNSR0IArs4c6QAAAAZiS0dEAP8A/wD/oL2nkwAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9wKExQZLWTEaOUAAAAddEVYdENvbW1lbnQAQ3JlYXRlZCB3aXRoIFRoZSBHSU1Q72QlbgAAAdpJREFUKM9tkL+L2nAARz9fPZNCKFapUn8kyI0e4iRHSR1Kb8ng0lJw6FYHFwv2LwhOpcWxTjeUunYqOmqd6hEoRDhtDWdA8ApRYsSUCDHNt5ul13vz4w0vWCgUnnEc975arX6ORqN3VqtVZbfbTQC4uEHANM3jSqXymFI6yWazP2KxWAXAL9zCUa1Wy2tXVxheKA9YNoR8Pt+aTqe4FVVVvz05O6MBhqUIBGk8Hn8HAOVy+T+XLJfLS4ZhTiRJgqIoVBRFIoric47jPnmeB1mW/9rr9ZpSSn3Lsmir1fJZlqWlUonKsvwWwD8ymc/nXwVBeLjf7xEKhdBut9Hr9WgmkyGEkJwsy5eHG5vN5g0AKIoCAEgkEkin0wQAfN9/cXPdheu6P33fBwB4ngcAcByHJpPJl+fn54mD3Gg0NrquXxeLRQAAwzAYj8cwTZPwPH9/sVg8PXweDAauqqr2cDjEer1GJBLBZDJBs9mE4zjwfZ85lAGg2+06hmGgXq+j3+/DsixYlgVN03a9Xu8jgCNCyIegIAgx13Vfd7vdu+FweG8YRkjXdWy329+dTgeSJD3ieZ7RNO0VAXAPwDEAO5VKndi2fWrb9jWl9Esul6PZbDY9Go1OZ7PZ9z/lyuD3OozU2wAAAABJRU5ErkJggg==" alt="[LOGO]"></a>
</div></footer>
</div>
</body>
</html>
